frontend/src/components/explorer/MapViewer.vue

```diff
@@ -350,7 +339,7 @@ export default {
       }
 
       if (!this.checkValidRequest(this.currentDisplayedType, this.currentDisplayedName)) {
-        this.showMessage(messages.mapNotFound, 'info');
+        this.showMessage(`Invalid map ID "${this.currentDisplayedName}"`, 'info');
```

Comment:
For consistency, this message should use the same `danger` class as above, in which case there should be no need for `loadErrorTypeMessage`.